charts/prometheus/templates/prometheus-auth-sidecar-configmap.yaml

```
@@ -1,5 +1,5 @@
 ################################
```

Nitpick, but while we're cleaning up these three comments, can we make the hash lines consistent?
```suggestion
#################################
```